src/eradius_client.erl

```diff
@@ -31,8 +34,8 @@
                         Req#radius_request.cmd == 'coareq' orelse
                         Req#radius_request.cmd == 'discreq')).
 
--type nas_address() :: {string() | binary() | inet:ip_address(), 
-                        eradius_server:port_number(), 
+-type nas_address() :: {string() | binary() | inet:ip_address(),
```

Comment:
Perhaps it is possible to do a separate commit with white space changes?

Reply:
This was done by `atom` editor

Reply:
It is okay (for me) to let atom editor move white space around. But please, in a separate pull request.